kubernetes/helm_charts/monitoring/alertrules/templates/promrulesProcess.yml

```diff
@@ -53,7 +53,7 @@ spec:
         message: {{`'Number of running processes are: {{$value}}'`}}
         summary: More than one process running
     - alert: secor_process_not_running_critical
-      expr: namedprocess_namegroup_states{groupname="secor",state="Sleeping"} != 13
+      expr: namedprocess_namegroup_states{groupname="secor",state="Sleeping"} != 15
```

Comment:
Make the count as a variable and update that in private repo

Reply:
updated the changes.